### ISSUE TITLE
NRE if user has not roles

### ISSUE THREAD

**Issue by dmitrydvm** [Nordes/IdentityServer4.LdapExtension#22]
If user has not roles, exception throw here https://github.com/Nordes/IdentityServer4.LdapExtension/blob/7447e522ab34b37feb518635587b153a42dba59a/IdentityServer.LdapExtension/UserModel/OpenLdapAppUser.cs#L63

How can I get user roles from AD?

**Comment by Nordes** [comment]
Yes, it will throw. I tested with MS AD, and in most of my cases I had a default role assigned to the user. A try/catch should be done in that part in order to avoid such exception. I will mark this as a bug to be fixed. I will try to find when I have the time to fix it.

The same issue will happen in MS AD

https://github.com/Nordes/IdentityServer4.LdapExtension/blob/7447e522ab34b37feb518635587b153a42dba59a/IdentityServer.LdapExtension/UserModel/ActiveDirectoryAppUser.cs#L67

**Comment by abhisheknaik07** [comment]
This is not yet fixed.
